test/command_callback/test_phpmd_command_callbacks.vader

```diff
@@ -10,3 +10,18 @@ Execute(Custom executables should be used for the executable and command):
   AssertLinter 'phpmd_test',
   \ ale#Escape('phpmd_test')
   \ . ' %s text cleancode,codesize,controversial,design,naming,unusedcode --ignore-violations-on-exit %t'
+
+Execute(User provided options should be used):
+  let g:ale_php_phpmd_suffixes = 'module,inc'
+
+  AssertLinter 'phpmd',
+  \ ale#Escape('phpmd')
+  \ . ' %s text cleancode,codesize,controversial,design,naming,unusedcode --suffixes module,inc --ignore-violations-on-exit %t'
+
+Given php(An empty PHP file):
+Execute(Current file extension should be used as the suffix):
+  let extension = expand('#' . bufnr('') . ':e')
+
+  AssertLinter 'phpmd',
+  \ ale#Escape('phpmd')
+  \ . ' %s text cleancode,codesize,controversial,design,naming,unusedcode --suffixes ' . extension . ' --ignore-violations-on-exit %t'
```

Comment:
Why is `--suffixes` used here? I assume that if the option wasn't set that it wouldn't be used at all?

Reply:
This PR is to add --suffixes if the user provides a list of suffixes with `ale_php_phpmd_suffixes`. If they didn't set that variable but the current file is a php filetype, then -suffixes will be set with the extension of the current file. That is what this test case is testing. Only when the user doesn't provide `ale_php_phpmd_suffixes` and the current file isn't a php filetype will it not add a `--suffixes` check.